updater.py

```diff
@@ -92,6 +100,14 @@ def notify_users_and_groups(message):
     for gid in loadjson("grouplist").keys():
         try:
             bot.send_message(gid, message, parse_mode="Markdown")
+        except telebot.apihelper.ApiException as ex:
+            GROUP_BLOCK_ERROR = "Forbidden: bot was kicked from the group chat"
```

Comment:
Have you tested this?

Reply:
I tested the concept on a seperate bot, unforunitely I can't test it on the actual bot.
Should work though.

Reply:
A separate bot is fine, as long as you tested the exact same scenario. Ok cool, merging this

Reply:
Alright, let me know if anything unexpected happens. Will fix ASAP.